Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -561,6 +678,214 @@ def use_homebrew_curl?(formula_or_cask, url)
       homebrew_curl_root_domains.include?(url_root_domain)
     end
 
+    # Identifies the latest version of the resource in a given Formulae/Casks and returns a Hash containing
+    # the version information for a resource. Returns a nil value if a latest version
+    # couldn't be found for a given resource.
+    sig {
+      params(
+        resource:  T.any(Resource),
+        json:      T::Boolean,
+        full_name: T::Boolean,
+        verbose:   T::Boolean,
+        debug:     T::Boolean,
+      ).returns(T.nilable(Hash))
+    }
+    def resource_version(
+      resource,
+      json: false,
+      full_name: false,
+      verbose: false,
+      debug: false
+    )
+      has_livecheckable = resource.livecheckable?
+
+      if debug
+        puts <<~EOS
+
+        EOS
+        odebug "Resource:          #{resource_name(resource, full_name: full_name)}"
+        odebug "Livecheckable?:    #{has_livecheckable ? "Yes" : "No"}"
```

Comment:
```suggestion
        puts "Resource:          #{resource_name(resource, full_name: full_name)}"
        puts "Livecheckable?:    #{has_livecheckable ? "Yes" : "No"}"
```

This output should align with the existing style seen in `#latest_version` (namely, using `puts` here instead of `odebug`).

Reply:
Okay ✅